### ISSUE TITLE
awlsimhw_linuxcnc: LinuxCNC E-Stop shall be triggered, if a fatal Awlsim exception occurs

### ISSUE THREAD

**Issue by mbuesch** [mbuesch/awlsim#21]
An attempt to trigger LinuxCNC E-Stop shall be made, if fatal exceptions occur in Awlsim.
Suggested by #20

**Comment by uncle-yura** [comment]
If the project has an error when starting linuxcnc, the server shuts down. If you work on a remote machine, you will not be able to overwrite the project.

```
awlsim-linuxcnc: [0.323] Awlsim error:

  Multiple definitions of FC 1.

Triggered LinuxCNC E-Stop
awlsim-linuxcnc: [0.911] Shutting down.
awlsim-linuxcnc: [0.912] LinuxCNC HAL module shutdown.
```
If you write the project with an error to an already running server, then everything works as usual.

**Comment by mbuesch** [comment]
That's a good point. The server should not shut down if there's something wrong with the project.

**Comment by mbuesch** [comment]
Implemented in Awlsim 0.72.0.
